ngraph/python/tests/test_ngraph/test_create_op.py

```diff
@@ -19,10 +19,9 @@
 
 import ngraph as ng
 import ngraph.opset1 as ng_opset1
+import ngraph.opset5 as ng_opset5
```

Comment:
This should be `opset6` as I assume.

Reply:
`import ngraph as ng `
I guess `ng `it's already `opset6 `version
I want to test the previous one.

Reply:
Ok, I see what is the case. Proper tests for opset6 are hidden below. Resolved.